src/Lesson_6/Cat.java

```diff
@@ -0,0 +1,34 @@
+package Lesson_6;
+
+public class Cat extends Animal{
+    private int MaxRun=200;
+    private double MaxJump=2;
+    private boolean MaxSwim=false;
+
+    public Cat(String name, int age,int run,double jump,int swim) {
+        super(name, age, run, jump, swim);
+    }
+    public boolean maxRun(){
+        if(getRun()<=MaxRun) return true;
+
+        return false;
```

Comment:
```suggestion
        return getRun()<=MaxRun;
```
также данный метод должен принимать параметр величины препятствия, с которым и делаем сравнение характеристик. Аналогично и для остальных методов